ballerina/websocket_sync_client.bal

```diff
@@ -253,13 +255,28 @@ public type CommonClientConfiguration record {|
     http:Cookie[] cookies?;
     ClientAuthConfig auth?;
     PingPongService pingPongHandler?;
+    WebSocketRetryConfig? retryConfig = ();
 |};
 
 # Configures the SSL/TLS options to be used for WebSocket client.
 public type ClientSecureSocket record {|
     *http:ClientSecureSocket;
 |};
 
+# Retry configurations for WebSocket.
+#
+# + maxCount - The maximum number of retry attempts. If the count is zero, the client will retry indefinitely.
+# + interval - The number of seconds to delay before attempting to reconnect.
+# + backOffFactor - The rate of increase of the reconnect delay. Allows reconnect attempts to back off when problems
+#                persist.
+# + maxWaitInterval - Maximum time of the retry interval in seconds.
+public type WebSocketRetryConfig record {|
+    int maxCount = 0;
+    int interval = 1;
```

Comment:
Interval should be in decimal type

Reply:
Fixed with https://github.com/ballerina-platform/module-ballerina-websocket/pull/303/commits/6db591d8501ae3b57c3b79d2b078b3e0e5685bd5